src/Client.php

```diff
@@ -59,7 +59,7 @@ public function __construct($instance, $username, $password)
     protected function client()
     {
         return $this->client ?? new \GuzzleHttp\Client([
-            'base_uri' => $this->instance . '/',
```

Comment:
Moest dit er niet achter? Staat mij iets bij dat het anders niet werkte?

Reply:
Nope, works fine. In fact, this was adding an additional slash.